tests/spicy/types/unit/skip-unit.spicy

```diff
@@ -0,0 +1,13 @@
+# @TEST-EXEC: printf 'x' | spicy-driver %INPUT >output
```

Comment:
Suggest to default to debug mode for usually faster compile times. Also we can drop `output` since it is empty and we do not look at it.

```suggestion
# @TEST-EXEC: printf 'x' | spicy-driver -d %INPUT
```